README.md

```diff
@@ -53,6 +53,18 @@ cargo run [-- CONFIG.json]
 
 for config see [this](./ricochet-headarches.json) and [that](./ricochet.json) as example
 
+# set logs at runtime
+
+`"scope"` is an array of what you want to debugs, likes `["nets", "vms"]`
```

Comment:
it would be clearer to say:
`"scope"` is an array of what resources you want to log, like `["nets", "vms"]`